app/src/main/java/com/li/weatherapp/utils/AirPollutionUtils.kt

```diff
@@ -22,4 +22,15 @@ object AirPollutionUtils {
         in 150..249 -> context.resources.getString(R.string.text_very_harmful_air_description)
         else -> context.resources.getString(R.string.text_dangerous_air_description)
     }
+
+    fun getAirPollutionColor(context: Context, aqiDegree: Int) = when (aqiDegree) {
+        in 0..19 -> getColor(context, R.color.chartreuse)
+        in 20..49 -> getColor(context, R.color.coral)
+        in 50..99 -> getColor(context, R.color.scarlet)
+        in 100..149 -> getColor(context, R.color.rose)
+        in 150..249 -> getColor(context, R.color.shocking_pink)
+        else -> getColor(context, R.color.electric_violet)
+    }
+
+    private fun getColor(context: Context, color: Int) = context.resources.getColor(color)
```

Comment:
thử hàm lồng hàm xem sao. tôi cx k bt ok k nh thử fun tý 
```suggestion
    fun getAirPollutionColor(context: Context, aqiDegree: Int) : Boolean  {
    return  when (aqiDegree) {
        in 0..19 -> getColor( R.color.chartreuse)
        in 20..49 -> getColor( R.color.coral)
        in 50..99 -> getColor( R.color.scarlet)
        in 100..149 -> getColor( R.color.rose)
        in 150..249 -> getColor( R.color.shocking_pink)
        else -> getColor( R.color.electric_violet)
        }
        fun getColor(color: Int) = context.resources.getColor(color)
    }

    
```

Reply:
t thử rồi. K được đâu ô